httpcore/_backends/auto.py

```diff
@@ -34,8 +34,11 @@ async def open_tcp_stream(
         port: int,
         ssl_context: Optional[SSLContext],
         timeout: TimeoutDict,
+        local_addr: Optional[bytes],
     ) -> AsyncSocketStream:
-        return await self.backend.open_tcp_stream(hostname, port, ssl_context, timeout)
+        return await self.backend.open_tcp_stream(
+            hostname, port, ssl_context, timeout, local_addr
```

Comment:
I reckon we ought to use `.open_tcp_stream(..., local_addr=local_addr) ` here.
Just makes it super obvious visually that it's an optional extra.

Reply:
Fixed.